Added OS-independent uninstaller

whokilleddb opened the pull request:
## Description
### Added `uninstaller.py`

The current `uninstaller.sh` script is specific to Debian/Ubuntu based distributions. This PR adds `uninstaller.py`: an OS-Independent script which uninstalls `pygoat` from your distribution regardless of the underlying Operating System.

### Tested On:
The script has been tested on the following platforms:
- Windows 10
- Arch Linux
- Debian Bullseye

adeyosemanputra reviewed (changes requested):
please let dockerfile. please remove dockerfile in your PR

whokilleddb commented:
Hi @adeyosemanputra I updated the `Dockerfile` because it had some redundant steps and added some dependencies:

## The changes explained:
- Added the following lines:
```bash
RUN apt-get -y install libpq-dev 
RUN apt-get -y install python3-dev
```
This is because during installation of pip dependencies, it sometimes fails during the installation of `pycogp2`(which is installed by one of the main dependencies). Thus, to prevent the same, the two dependencies have been added to add stability to the build process. [See the stackoverflow answer [here](https://stackoverflow.com/questions/5420789/how-to-install-psycopg2-with-pip-on-python)]

- Also, the original Dockerfile had the following steps:
```bash
COPY requirements.txt requirements.txt
RUN pip install -r requirements.txt
# copy project
COPY . /app/
```
Since while copying the application source files in the last step  automatically copies `requirements.txt`, it makes the statement `COPY requirements.txt requirements.txt` obsolete. Hence, this reduces a step during the build process.

Let me know if you would still like me to revert to the original `Dockerfile` and I would be happy to make the changes once you have reviewed the changes.
